hudi-spark-datasource/hudi-spark3-common/src/main/java/org/apache/hudi/spark3/internal/ReflectUtil.java

```diff
@@ -29,17 +27,10 @@ public class ReflectUtil {
   public static DateFormatter getDateFormatter(ZoneId zoneId) {
```

Comment:
should we bother getting rid of this param?

Reply:
We should have a follow-up PR to clean up any logic that is no long needed for Spark 3.3 and above.

Reply:
https://issues.apache.org/jira/browse/HUDI-8086